scylla-cql/src/types/deserialize/value.rs

```diff
@@ -1656,10 +1651,18 @@ impl Display for BuiltinDeserializationErrorKind {
             BuiltinDeserializationErrorKind::SetOrListError(err) => err.fmt(f),
             BuiltinDeserializationErrorKind::MapError(err) => err.fmt(f),
             BuiltinDeserializationErrorKind::TupleError(err) => err.fmt(f),
+            BuiltinDeserializationErrorKind::CustomTypeNotSupported(typ) => write!(f, "Support for custom types is not yet implemented: {}", typ),
         }
     }
 }
 
+impl From<LowLevelDeserializationError> for BuiltinDeserializationErrorKind {
+    #[inline]
+    fn from(err: LowLevelDeserializationError) -> Self {
+        Self::LowLevelDeserializationError(err)
+    }
+}
+
```

Comment:
In other places we are using `thiserror` crate to generate the conversions. Why not do the same here?